src/Sample.Worker/Program.cs

```diff
@@ -48,6 +54,19 @@ public static IHostBuilder CreateHostBuilder(string[] args)
                                 e.ConfigureConsumer<OrderSubmittedConsumer>(context);
                             });
 
+                            cfg.Publish<OrderShippedBase>(configurator => configurator.Exclude = true);
+                            cfg.SubscriptionEndpoint<OrderShipped>("order-shipped-consumer", e =>
+                            {
+                                e.ConfigureConsumeTopology = false;
+                                e.AutoStart = true;
+                                e.AutoDeleteOnIdle = TimeSpan.FromDays(30);
+                                e.MaxDeliveryCount = 2; 
+                                e.RequiresSession = true;
+                                e.MaxConcurrentCalls = 8; //like AZ Function default. 8 Concurrent consumers handling 1 ServiceBus Session
+                                e.PrefetchCount = 0;
```

Comment:
Not really a valid value you should specify.

Reply:
I guess you are right, it isn't needed AZ funcs prefect too with peek, as long as the prefect does not ack and respects sessions.
I believe it does, yes?